### ISSUE TITLE
Exceptions is not thrown

### ISSUE THREAD

**Issue by GazziFX** [kekyo/Lepracaun#1]
```csharp
using var app = new Application();

app.Run(async () =>
{
    throw new NotImplementedException();
});
```

Just quits without any messages

**Comment by kekyo** [comment]
@GazziFX Oh, thanks reporting,

This was caused by an `UnhandledException` event that could receive an occurrence, but if no one handled this event, the event was ignored. This will be fixed soon.

**Comment by kekyo** [comment]
Fixed it and being deployed 1.1.0, could you try it?

**Comment by GazziFX** [comment]
Yes, it works, thanks!
